query_test.go

```diff
@@ -54,10 +54,10 @@ type testRow struct {
 }
 
 type testRowPtr struct {
-	A_P *int64   `db:"a_p,omitempty"`
-	B_P *string  `db:"b_p,omitempty"`
-	C_P *string  `db:"c_p,omitempty"`
-	D_P *float64 `db:"d_p,omitempty"`
+	AP *int64   `db:"a_p,omitempty"`
+	BP *string  `db:"b_p,omitempty"`
+	CP *string  `db:"c_p,omitempty"`
+	DP *float64 `db:"d_p,omitempty"`
```

Comment:
Please leave the "_", this shows clearly that it is a pointer.

may be "pA", "pB" ?

Reply:
Really, It doesn't mind. If you think it's better, I can update the commit.

Reply:
You can't Use unexported fields with struct tags. But I can rename them to PointerA to be readable enough.

Reply:
Lets do "Ap", "Bp" then.